### ISSUE TITLE
<form>: HTML looks broken

### ISSUE THREAD

**Issue by alrdytaken** [w3c/html#1631]
Looks like the HTML code is broken. Notice the dd&gt;
https://www.w3.org/TR/html52/sec-forms.html#the-form-element

```
<dt><a data-link-type="dfn" href="dom.html#allowed-aria-role-attribute-values" id="ref-for-allowed-aria-role-attribute-values⑦⑧">Allowed ARIA role attribute values</a>:
       dd&gt;<code>form</code> […]
```

**Comment by scottaohara** [comment]
Thank you @alrdytaken.

Looks like an issue specific to the 5.2 spec, as the HTML is fixed in the 5.3 draft.

**Comment by nschonni** [comment]
Fixed in the 5.2 branch, but I'm not sure how/if it gets published

**Comment by siusin** [comment]
Thanks all. HTML 5.3 will be published as a WG Note soon.

We're closing this issue on the W3C HTML specification because the W3C and WHATWG are now [working together on HTML](https://www.w3.org/blog/2019/05/w3c-and-whatwg-to-work-together-to-advance-the-open-web-platform/), and all issues are being discussed on the [WHATWG repository](https://github.com/whatwg/html).

If you filed this issue and you still think it is relevant, please open a [new issue](https://github.com/whatwg/html/issues) on the WHATWG repository and reference this issue (if there is useful information here). Before you open a new issue, please check for existing issues on the WHATWG repository to avoid duplication.

If you have questions about this, please open an issue on the [W3C HTML WG repository](https://github.com/w3c/htmlwg/issues/new) or send an email to public-html@w3.org.
